src/main/default/lwc/lookup/lookup.js

```diff
@@ -46,8 +46,10 @@ export default class Lookup extends NavigationMixin(LightningElement) {
     // PUBLIC FUNCTIONS AND GETTERS/SETTERS
     @api
     set selection(initialSelection) {
-        this._curSelection = Array.isArray(initialSelection) ? initialSelection : [initialSelection];
-        this.processSelectionUpdate(false);
+        if (initialSelection && initialSelection.length > 0) {
```

Comment:
`initialSelection.length` assumes that `initialSelection` is an array but we also support other types.
Please change to:
```js
if (initialSelection)
```

Reply:
Will do, thanks.